### PR TITLE
HOTFIX  refresh importance bug

### PR DESCRIPTION
**Pull request by domakun**

### CODE REVIEW

**Comment by domakun** [comment]
> when clicking on the refresh button
> image

fixed

File: src/views/wallet/wallet-details/WalletDetailsTs.ts [hunk 1]
```diff
@@ -109,6 +110,19 @@ export class WalletDetailsTs extends Vue {
         }
     }
 
+    refreshImportance(){
+      const {importanceRefreshTimestamp, wallet} = this
+      const currentTimestamp = new Date().valueOf()
+      if (currentTimestamp - importanceRefreshTimestamp <= 2000) {
+        this.$Notice.destroy()
```
Comment (evias) [inline src/views/wallet/wallet-details/WalletDetailsTs.ts]: why does it have to destroy other notices?

Comment (domakun) [reply src/views/wallet/wallet-details/WalletDetailsTs.ts]: This is a function to clear the Notice component of the page, which can make the page more tidy.

**Comment by domakun** [comment]
will craete a new refresh importance pr

**Pull request closed without merging**